os/include/tinyara/sched.h

```diff
@@ -86,18 +86,24 @@
 /********************************************************************************
  * Pre-processor Definitions
  ********************************************************************************/
-#if defined(CONFIG_APP_BINARY_SEPARATION) && defined(CONFIG_ARM_MPU)
+#if defined(CONFIG_ARM_MPU) && (defined(CONFIG_APP_BINARY_SEPARATION) || defined (CONFIG_MPU_STACKGUARD))
+#define REGS_PER_REGION 3
```

Comment:
One question.
Because the enum structure has 3 elements, is this defined as 3?
If it is,
```
enum {
    REG_RNR = 0,
    REG_RBAR,
    REG_RASR,
    REG_MAX
};
```
let's use `REG_MAX` instead of new definition. I don't want to add duplicated manual definition .

Reply:
changed